packages/libmpg123/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: libmpg123
```

Comment:
I was not aware of that. Emscripten does not seem to provide `.pc` files for these libraries. I can still use them if you want, but I think it's a little bit unclean because I now have to specify explicit flags to link, for example, `libsndfile` with `libflac` (just because `libflac` depends on `libogg` and `pkg-config` cannot resolve the dependencies anymore).